oozie/src/test/java/org/apache/falcon/oozie/feed/OozieFeedWorkflowBuilderTest.java

```diff
@@ -261,6 +266,54 @@ private void buildCoordAndValidateFrequency(final String expectedFrequency) thro
     }
 
     @Test
+    public void testPostProcessing() throws Exception{
+        StartupProperties.get().setProperty("falcon.postprocessing.enable", "false");
+        OozieEntityBuilder builder = OozieEntityBuilder.get(postFeed);
+        Path bundlePath = new Path("/projects/falcon/");
+        builder.build(trgCluster, bundlePath);
+        BUNDLEAPP bundle = getBundle(trgMiniDFS.getFileSystem(), bundlePath);
+        List<COORDINATOR> coords = bundle.getCoordinator();
+        COORDINATORAPP coord = getCoordinator(trgMiniDFS, coords.get(0).getAppPath());
+
+        WORKFLOWAPP workflow = getWorkflowapp(trgMiniDFS.getFileSystem(), coord);
+
+        Boolean userAction = false;
+        Boolean postProcessing = true;
+
+        coord = getCoordinator(trgMiniDFS, coords.get(1).getAppPath());
+
+        for(Object action : workflow.getDecisionOrForkOrJoin()){
+            if (action instanceof ACTION && ((ACTION)action).getName().equals("eviction")){
+                userAction = true;
+            }
+            if (action instanceof ACTION && ((ACTION)action).getName().contains("post")){
+                postProcessing = false;
+            }
+
+        }
+        assertTrue(userAction);
+        assertTrue(postProcessing);
+
+        workflow = getWorkflowapp(trgMiniDFS.getFileSystem(), coord);
+
+        userAction = false;
+        postProcessing = true;
+
+        for(Object action : workflow.getDecisionOrForkOrJoin()){
```

Comment:
You can iterate over the coords in a loop without having to repeat this code. You can look for either eviction or replication inside.